structure_tests/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+
+export VERSION=$1
+
+if [ -z "$1" ]; then
+  echo "Please provide valid JSON schema version to tag image."
```

Comment:
sorry, yep. this was wording left over from where i was actually using the JSON schema version in the image tag.